src/commands/commit/list.ts

```diff
@@ -88,6 +88,7 @@ export default class CommitList extends BaseCommand<typeof CommitList> {
       },
       commit_message: {
         header: "Commit message",
+        get: (entry) => entry.commit_message?.trim()
```

Comment:
Sometimes there are new lines after a given entry, which looks really bad in the terminal. After digging into this, I realized the solution is to delete all whitespaces in the commit message

Before:

<img width="1440" alt="Screen Shot 2023-11-07 at 13 49 35" src="https://github.com/knocklabs/knock-cli/assets/67347884/7c9656ba-faf1-4286-a2c6-323991bf01b2">

After:
<img width="1439" alt="Screen Shot 2023-11-07 at 13 49 53" src="https://github.com/knocklabs/knock-cli/assets/67347884/10b1852d-55d2-4ac7-aae2-a1b06c02449e">